scrapy/commands/parse.py

```diff
@@ -192,6 +192,9 @@ def callback(response):
             # parse items and requests
             depth = response.meta['_depth']
 
+            if response.request.callback:
```

Comment:
Hi,
I am new to the project and to open source development as well. but would you explain why 
if response.request.callback: check is neccessary. also If i am not wrong , you can just use cb and not request.meta['_callback']

Reply:
Hey @gekco , I believe there's a comment in the review changes that has more detail about this!